bambi/plots/create_data.py

```diff
@@ -0,0 +1,123 @@
+import pandas as pd
+import itertools
+
+import bambi as bmb
```

Comment:
I wouldn't import `bambi` in a submodule. It's better to import the things we need from it as we do with, for example, `from bambi.utils import clean_formula_lhs`

Reply:
Ahh good point. I was importing it for type hints. But you are right, in a submodule, I could do `from Bambi import Model` and use `Model`